server/routes/api.js

```diff
@@ -131,33 +128,44 @@ router.post('/contactFormSubmit', function(req, res){
 
 router.post('/corpEventFormSubmit', function(req, res){
   scrubHeaders(res)
-  var transporter = nodemailer.createTransport({
-    service: "Gmail",
-    auth: {
-        user: config.gmail.user_name,
-        pass: config.gmail.password
+  const gRecaptchaResponseValue = req.body.captcha
+  var secretKey = config.recaptcha.corpevent.secret_key;
+  const verifyUrl = `https://www.google.com/recaptcha/api/siteverify?secret=${secretKey}&response=${gRecaptchaResponseValue}&remoteip=${req.connection.remoteAddress}`;
+  request(verifyUrl, (err, response, body) => {
+    body = JSON.parse(body)
+    if(body.success !== undefined && !body.success){
+      return res.json({"error": false, "msg":"Failed captcha verification"});
     }
-  })
-  var mailOptions = {
-    from: config.gmail.from,
-    to: config.gmail.mailTo,
-    subject: 'New Corporate Event Form Submission from ' + req.body.firstName + ' ' + req.body.lastName, // Subject line
-    text:  `From: ${req.body.firstName} ${req.body.lastName}
-          \nEmail: ${req.body.email}
-          \nSubject: ${req.body.phone}
-          \nEvent Title: ${req.body.eventTitle}
-          \nStart Date: ${req.body.startDate}
-          \nEnd Date: ${req.body.endDate}
-          \nWebsite: ${req.body.website}
-          \nAdditional Info: ${req.body.additionalInfo}`
-  }
-  transporter.sendMail(mailOptions, function(error, info){
-    if(error){
-        res.json({yo: 'error'});
-    }else{
-        res.json({yo: info.response});
-    };
-  })
+    else if(body.success !== undefined && body.success) {
+      var transporter = nodemailer.createTransport({
+        service: "Gmail",
+        auth: {
+            user: config.gmail.user_name,
+            pass: config.gmail.password
+        }
+      })
+      var mailOptions = {
+        from: config.gmail.from,
+        to: config.gmail.mailTo,
+        subject: 'New Corporate Event Form Submission from ' + req.body.firstName + ' ' + req.body.lastName, // Subject line
+        text:  `From: ${req.body.firstName} ${req.body.lastName}
```

Comment:
This actually sends this as plain text, ja? just making sure vis-a-vis SMTP Injection or HTML injection

Reply:
Plain text.  I'll add extra validation.